package.json

```diff
@@ -49,6 +50,7 @@
     "semver": "^5.6.0",
     "svgo": "^1.1.1",
     "uglify-js": "^3.5.0",
+    "url": "^0.11.0",
```

Comment:
You shouldn't need this dependency as the 'url' module is part of Node. This library is only needed if running code in the browser.